projects/nuvoton/numaker_iot_m487_wifi/uvision/aws_demos/aws_demos.uvproj

```diff
@@ -1,2346 +1,3673 @@
-						</File>
-						<File>
-							<FileName>oid.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/oid.c</FilePath>
-						</File>
-						<File>
-							<FileName>padlock.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/padlock.c</FilePath>
-						</File>
-						<File>
-							<FileName>pem.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/pem.c</FilePath>
-						</File>
-						<File>
-							<FileName>pk.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/pk.c</FilePath>
-						</File>
-						<File>
-							<FileName>pk_wrap.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/pk_wrap.c</FilePath>
-						</File>
-						<File>
-							<FileName>pkcs12.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/pkcs12.c</FilePath>
-						</File>
-						<File>
-							<FileName>pkcs5.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/pkcs5.c</FilePath>
-						</File>
-						<File>
-							<FileName>pkparse.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/pkparse.c</FilePath>
-						</File>
-						<File>
-							<FileName>pkwrite.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/pkwrite.c</FilePath>
-						</File>
-						<File>
-							<FileName>platform.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/platform.c</FilePath>
-						</File>
-						<File>
-							<FileName>platform_util.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/platform_util.c</FilePath>
-						</File>
-						<File>
-							<FileName>ripemd160.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/ripemd160.c</FilePath>
-						</File>
-						<File>
-							<FileName>rsa.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/rsa.c</FilePath>
-						</File>
-						<File>
-							<FileName>rsa_internal.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/rsa_internal.c</FilePath>
-						</File>
-						<File>
-							<FileName>sha1.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/sha1.c</FilePath>
-						</File>
-						<File>
-							<FileName>sha256.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/sha256.c</Fi
```

Comment:
You are able to compile both **libraries\abstractions\secure_sockets\freertos_plus_tcp\iot_secure_sockets.c** AND **vendors/nuvoton/boards/numaker_iot_m487_wifi/ports/secure_sockets/iot_secure_sockets.c** at the same time? These have the same function names...?